docs/getting_started/faq.rst

```diff
@@ -0,0 +1,110 @@
+implementation. While we would be happy to donate parts of Modin that
+make sense in pandas, many of these components would require significant (or
+total) redesign of the pandas architecture. Modin's architecture goes beyond
+pandas, which is why the pandas API is just a thin layer at the user level. To learn
+more about Modin's architecture, see the architecture_ documentation.
+
+How much faster can I go with Modin compared to pandas?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Modin is designed to scale with the amount of hardware available.
+Even in a traditionally serial task like ``read_csv``, we see large gains by efficiently 
+distributing the work across your entire machine. Because it is so light-weight, 
+Modin provides speed-ups of up to 4x on a laptop with 4 physical cores. This speedup scales
+efficiently with larger machines with more cores. We have several published papers that
+include performance results and comparisons against pandas.
+
+How much more data would I be able to process with Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+We have focused heavily on bridging the solutions between DataFrames for small 
+data (e.g. pandas) and large data. Often data scientists require different tools 
+for doing the same thing on different sizes of data. The DataFrame solutions that 
+exist for 1MB do not scale to 1TB+, and the overheads of the solutions for 1TB+ 
+are too costly for datasets in the 1KB range. With Modin, because of its light-weight, 
+robust, and scalable nature, you get a fast DataFrame at 1MB and 1TB+.
+
+How does Modin work under the hood?
+""""""""""""""""""""""""""""""""""""
+Modin is logically separated into different layers that represent the hierarchy of a 
+typical Database Management System. User queries which perform data transformation, 
+data ingress or data egress pass through the Modin query compiler which translates 
+queries from the Pandas API Layer and sends them to the Modin Core DataFrame. The Modin
+Core DataFrame has an efficient dataframe partitioning schema which allows for extreme
+parallelization. From here, the Modin DataFrame works with task parallel frameworks like
+Ray or Dask to execute computation, and then return the results to the user.
+
+For more details, take a look at our system architecture_. 
+
+If I’m only using my laptop, can I still get the benefits of Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Absolutely! Unlike other parallel DataFrame systems, Modin is an extremely 
+light-weight, robust DataFrame. Because it is so light-weight, Modin provides 
+speed-ups of up to 4x on a laptop with 4 physical cores.
+
+How do I use jupyter/collab notebooks with Modin? 
+""""""""""""""""""""""""""""""""""""""""""""
+Just like you would use any other notebook, just replace your pandas import
+with modin:
+
+.. code-block:: python
+   # import pandas as pd
+   import modin.pandas as pd
```

Comment:
Add link to Colab instructions. Also link to notebook tutorials.

Reply:
BTW, this code-block seems to be isn't rendered for some reason. It even doesn't exist in the code of the HTML page, sphinx build logs may give a hint about the reasons.
![image](https://user-images.githubusercontent.com/62142979/145573567-6d48d608-083a-4a9b-b91b-55776fc364ab.png)